src/metadata.ts

```diff
@@ -47,6 +47,11 @@ export const updateMetadata = ({title, description, url, image}: {title?: string
     _setMeta('property', 'twitter:image:src', image);
   }
 
+  if (imageAlt) {
+    _setMeta('property', 'og:image:alt', imageAlt);
```

Comment:
`og:image:alt` is not mentioned in the spec (https://developers.facebook.com/docs/sharing/webmasters/#markup). In this case the `twitter:` specific tag makes sense.

Reply:
@keanulee http://ogp.me/#structured